corehq/util/soft_assert/api.py

```diff
@@ -87,20 +91,21 @@ def send_to_admins(subject, message):
             message=message,
         )
 
-    if to and notify_admins:
-        def send(info):
-            _send_message(info, backend=send_to_admins)
-            _send_message(info, backend=send_to_recipients)
-    elif to:
-        def send(info):
-            _send_message(info, backend=send_to_recipients)
-    elif notify_admins:
-        def send(info):
-            _send_message(info, backend=send_to_admins)
-    else:
+    def write_to_log_file(subject, message):
+        logger.debug(message)
```

Comment:
What do you think of increasing the logging level to WARNING or ERROR? (it is an assertion failure after all)